packages/opc-feedback/test/opc-feedback.test.ts

```diff
@@ -31,14 +31,14 @@ describe('opc-feedback', () => {
   });
 
   // Add more tests here
-  it('does have default url by default', async () => {
-    expect(opcFeedbackPanelElement.url).toEqual('/feedback');
+  it('does have default spaUrl by default', async () => {
+    expect(opcFeedbackPanelElement.spaUrl).toEqual('/feedback');
   });
 
-  it('can initiate url for spa', async () => {
-    opcFeedbackPanelElement.url = 'https://example.com';
+  it('can initiate spaUrl for spa', async () => {
+    opcFeedbackPanelElement.spaUrl = 'https://example.com';
     await opcFeedbackPanelElement.updateComplete;
-    expect(opcFeedbackPanelElement.url).toEqual('https://example.com');
+    expect(opcFeedbackPanelElement.spaUrl).toEqual('https://example.com');
   });
 
```

Comment:
can you add tests for the template variables?

Reply:
updated the tests